apply/screening-example.js

```diff
@@ -0,0 +1,118 @@
+import { readFileSync, writeFileSync } from 'fs';
+import { applyPlan, simpleResolver } from 'encender';
+
+// PlanDefintion resources
+const screeningPlan = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/PlanDefinition-CervicalCancerScreening.json'));
+const symptomaticScreeningPlan = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/PlanDefinition-CervicalCancerScreeningSymptomaticIndividuals.json'));
+const desExposureScreeningPlan = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/PlanDefinition-CervicalCancerScreeningDesExposure.json'));
+const immunocompromisedScreeningPlan = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/PlanDefinition-CervicalCancerScreeningImmunocompromisedIndividuals.json'));
+const averageRiskScreeningPlan = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/PlanDefinition-CervicalCancerScreeningAverageRisk.json'));
+
+// ActivityDefinition resources
+const cytologyActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-CervicalCytologyScreeningRecommendation.json'));
+const colposcopyActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-ColposcopyActivityDefinition.json'));
+const errorActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-CommunicateErrors.json'));
+const communicateActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-CommunicateUnstructuredRecommendations.json'));
+const cotestActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-CotestingScreeningRecommendation.json'));
+const dashboardActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-DisplayCervicalCancerMedicalHistory.json'));
+const hpvActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-PrimaryHpvScreeningRecommendation.json'));
+const surveillanceActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-SurveillanceActivityDefinition.json'));
+const treatmentActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-TreatmentActivityDefinition.json'));
+
+// Questionnaire resources
+const provideMoreInformationQuestionnaire = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/Questionnaire-ProvideMoreInformation.json'));
```

Comment:
Could we just have a loop that brings in all resources from `fsh-tank/fsh-generated/resources/` at once? If for some reason they can't, even making some sort of path variable could be more DRY and less cumbersome.